prover/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/iden3/go-rapidsnark/prover
+
+go 1.18
+
+replace github.com/iden3/go-rapidsnark/types => ../types
```

Comment:
I hope this is a temporary solution, and we will remove this line after tagging accurate versions